tests/settings.py

```diff
@@ -6,7 +6,7 @@
 DEPLOYMENTS = f"{BASEDIR}/deployments"
 PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 TEST_DATA = f"{PROJECT_ROOT}/data"
-DEFAULT_IMAGE = "nginx/nginx-ingress:edge"
+DEFAULT_IMAGE = "nginx/nginx-ingress:1.9.0"
```

Comment:
I would double check with @vepatel if updating test related files is necessary

Reply:
I'd say It's fine for consistency purposes.